### PR TITLE
[INLONG-755]Go SDK Consumer Result

### PR DESCRIPTION
**Pull request by TszKitLo40**
Signed-off-by: Zijie Lu <wslzj40@gmail.com>

### CODE REVIEW

**Comment by TszKitLo40** [comment]
@charlely @gosonzhang PTAL

**Review by gosonzhang — APPROVED** [review]
+1

File: tubemq-client-twins/tubemq-client-go/client/consumer_impl.go [hunk 1]
```diff
@@ -383,7 +383,7 @@ func (c *consumer) processRebalanceEvent() {
 }
 
 func (c *consumer) disconnect2Broker(event *metadata.ConsumerEvent) {
-	log.Tracef("[disconnect2Broker] connect event begin, client id=%s", c.clientID)
+	log.Tracef("[disconnect2Broker] connect event begin, client=%s", c.clientID)
```
Comment (charlely) [inline tubemq-client-twins/tubemq-client-go/client/consumer_impl.go:386]: This is disconnect function, but this is concet event.

File: tubemq-client-twins/tubemq-client-go/client/consumer_impl.go [hunk 2]
```diff
@@ -393,7 +393,7 @@ func (c *consumer) disconnect2Broker(event *metadata.ConsumerEvent) {
 		}
 	}
 	event.SetEventStatus(metadata.Disconnect)
-	log.Tracef("[disconnect2Broker] connect event finished, client id=%s", c.clientID)
+	log.Tracef("[disconnect2Broker] connect event finished, client=%s", c.clientID)
```
Comment (charlely) [inline tubemq-client-twins/tubemq-client-go/client/consumer_impl.go:396]: dito

**Review by charlely — APPROVED** [review]

**Pull request merged**